src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1598,31 +1630,42 @@ bool Graph::InsertNode(EdgePtr edge, NodePtr node, bool initNode) {
     return InsertNode(edge->getParent(), edge->getChild(), node, iIndex, oIndex, initNode);
 }
 
+void Graph::AddNodes(const std::vector<NodePtr>& new_nodes, bool initNode) {
+    if (initNode) {
+        for (auto& node : new_nodes) {
+            node->getSupportedDescriptors();
+            node->initSupportedPrimitiveDescriptors();
+            node->filterSupportedPrimitiveDescriptors();
+            node->selectOptimalPrimitiveDescriptor();
+        }
+        for (auto& node : new_nodes) {
+            resolveInPlaceDirection(node);
+        }
+        for (auto& node : new_nodes) {
+            node->initOptimalPrimitiveDescriptor();
+        }
+    }
+    for (auto& node : new_nodes) {
+        graphNodes.push_back(node);
+    }
+    node->updateConstantType();
```

Comment:
node is unavailable in this scope. did you mean smth else?

Reply:
Sorry, fixed.